duckduckgo/content-scope-scripts
# messaging: page-world bridge

**Asana Task/Github Issue:** https://app.asana.com/0/1201141132935289/1208740665800207/f

## Description

- [x] added a new feature called `messageBridge` (apple-isolated only for now)
- [x] added `createMessageBridge` on `navigatorInterface` - to return the MessagingInterface
   -  this is always present, but will throw an exception if the bridge is not allowed, installed or enabled
   - exception is just a generic message, it does not reveal a reason

## Testing Steps

- See the macOS PR for where I implemented this E2E https://github.com/duckduckgo/macos-browser/pull/3558

## Checklist


*Please tick all that apply:*
